mule-packager/src/main/java/org/mule/tools/api/util/DependencyProject.java

```diff
@@ -16,22 +16,21 @@
 import org.mule.maven.client.api.model.BundleDescriptor;
 import org.mule.maven.client.api.model.BundleScope;
 import org.mule.tools.api.classloader.model.ArtifactCoordinates;
-import org.mule.tools.api.util.Project;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class DependencyProject implements Project {
 
-  private MavenProject mavenProject;
+  private final MavenProject mavenProject;
 
   public DependencyProject(MavenProject mavenProject) {
     this.mavenProject = mavenProject;
   }
 
   @Override
   public List<ArtifactCoordinates> getDependencies() {
-    return mavenProject.getDependencies().stream().map(ArtifactUtils::toArtifactCoordinates).collect(Collectors.toList());
+    return mavenProject.getArtifacts().stream().map(ArtifactUtils::toArtifactCoordinates).collect(Collectors.toList());
```

Comment:
Maybe we should create a new method. Because this one is used here https://github.com/mulesoft/mule-maven-plugin/blob/4d76fb6c0b841fb55751a036c7f6f04b7d838750/mule-packager/src/main/java/org/mule/tools/api/validation/resolver/model/DependenciesFilter.java#L41 and there it's supposed to return only direct deps

Reply:
👍